toc.json

```
@@ -78,24 +78,30 @@
           "href": "/api-docs/storefront/graphql/graphql-storefront-api-overview",
           "uri": "docs/api-docs/storefront/graphql/graphql-api-overview.md"
         },
+        {
+          "type": "item",
+          "title": "Faceted and Textual Search",
+          "href": "/api-docs/storefront/graphql/search",
+          "uri": "docs/api-docs/storefront/graphql/graphql-faceted-textual-search.md"
+        },
         {
           "type": "item",
           "title": "Code Samples",
           "href": "/api-docs/storefront/graphql/graphql-storefront-api-samples",
           "flag": "code samples",
           "uri": "docs/api-docs/storefront/graphql/graphql-api-examples.md"
         },
+        // {
+        //  "type": "item",
+        //  "title": "Storefront GraphQL Explorer",
+        //  "href": "",
+        //  "uri": ""
+        // },
         {
```

```suggestion
        {
          "type": "item",
          "title": "GraphQL Storefront Explorer",
          "href": "/api-docs/storefront/graphql/explorer",
          "uri": "https://developer.bigcommerce.com/graphql-explorer"
        },
        {

```

explorer is back up!